crypto/keyring/keyring.go

```diff
@@ -110,6 +110,10 @@ type Exporter interface {
 	// Export public key
 	ExportPubKeyArmor(uid string) (string, error)
 	ExportPubKeyArmorByAddress(address sdk.Address) (string, error)
+
+	// ExportPrivKeyHex returns a private key in unarmored hex format
+	ExportPrivKeyHex(uid string) (string, error)
```

Comment:
I'd create a separate interface for this function (e.g. `UnsafeExporter`), so that we don't force clients to implement something that they may consider insecure. I'll propose some changes in another PR